sonar-kotlin-plugin/src/test/java/org/sonarsource/kotlin/verifier/KotlinVerifier.kt

```diff
@@ -48,7 +48,7 @@ private val DEFAULT_TEST_JARS_DIRECTORY = "../kotlin-checks-test-sources/build/t
 class KotlinVerifier(private val check: AbstractCheck) {
 
     var fileName: String = ""
-    var classpath: List<String> = System.getProperty("java.class.path").split(":") + DEFAULT_KOTLIN_CLASSPATH
+    var classpath: List<String> = System.getProperty("java.class.path").split(System.getProperty("path.separator")) + DEFAULT_KOTLIN_CLASSPATH
```

Comment:
Is this for portability? Or does it help this specific PR? Anyway, it is probably a good change

Reply:
Yea, it is for portability (on Windows, splitting by `:` is actually completely breaking the classpath - you need to use `;` instead). To be honest, I am surprised our unit tests all passed on Windows so far :thinking: